### ISSUE TITLE
Error not working

### ISSUE THREAD

**Issue by shrivi34** [Orelsaada/YouTube-Subpals-bot#2]
it says iframe not responding in 5 seconds. and says selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"[name="channelid"]"
screenshot:(https://user-images.githubusercontent.com/27879698/78890173-c5bc2380-7a82-11ea-9d97-8d15fcfbdbba.PNG)

**Comment by Orelsaada** [comment]
They're changing the iframes very often, for many reasons.
Need to find the right iframe again.

**Comment by Orelsaada** [comment]
Did you register to the site before running the script?

**Comment by shrivi34** [comment]
Yes ,I have registered before. 
Now there is different error. 
It says:
 "Refused to display 'https://accounts.google.com/ServiceLogin?uilel=3&service=youtube&passive=true&hl=en-GB&continue=https%3A%2F%2Fwww.youtube.com%2Fsignin%3Faction_handle_signin%3Dtrue%26feature%3Dpassive%26app%3Ddesktop%26hl%3Den-GB%26next%3D%252Fsignin_passive' in a frame because it set 'X-Frame-Options' to 'deny'.", source: https://www.youtube.com/watch?v=qletMHSAOII (0)"

Screenshot: 
https://user-images.githubusercontent.com/27879698/79222748-bcadc680-7e75-11ea-98b0-6b7f0d649531.PNG

**Comment by shrivi34** [comment]
Stack exchange says: 
"You may also replace watch?v= with embed/ so http://www.youtube.com/watch?v=eCfDxZxTBW4 becomes http://www.youtube.com/embed/eCfDxZxTBW4"
But how do we do that?

**Comment by Orelsaada** [comment]
Do you entered your gmail account credentials?
Do you have on your mail account 2 factor authentication? or maybe another special settings?

**Comment by shrivi34** [comment]
I did enter the credentials and no special settings are enabled.

**Comment by shrivi34** [comment]
can you mention the chrome,driver and selenium version you used

**Comment by Orelsaada** [comment]
Selenium: 3.141.0
Chrome driver: 2.38.552522

**Comment by Orelsaada** [comment]
Try to manually do 1 like & subscribe in Subpals.
https://www.subpals.com/

**Comment by shrivi34** [comment]
yeah the manual thing works,but not able to subscribe and like automatically.

**Comment by Orelsaada** [comment]
I just try with a new mail to do it, found out that google tries to identify you, so manually send code to my phone to verify google.

then ran the script again and it worked.

try also to allow "less secure apps" using your gmail : 
https://support.google.com/accounts/answer/6010255?hl=en

**Comment by yahiakortam** [comment]
I'm confused on how to use this. I just switched from mac to windows today. Can u explain it in more thorough instructions.

**Comment by LeandroLimaPRO** [comment]
#9

**Comment by LeandroLimaPRO** [comment]
#13
